R/robustbase-glmrob-tidiers.R

```diff
@@ -0,0 +1,108 @@
+#' @templateVar class glmrob
+#' @template title_desc_tidy
+#'
+#' @param x A `glmrob` object returned from [robustbase::glmrob()].
+#' @template param_confint
+#' @template param_unused_dots
+#'
+#' @evalRd return_tidy(regression = TRUE)
+#'
+#' @details For tidiers for robust models from the \pkg{MASS} package see
+#'   [tidy.rlm()]. For tidiers for robust models from the \pkg{robust} package
```

Comment:
Let's not mention `robust` here since `robust` isn't developed anymore and is largely broken so far as I can tell. Definitely don't want people using it.